### ISSUE TITLE
TXT file upload using UTF-16 LE Encoding

### ISSUE THREAD

**Issue by milesfrankland** [grokability/snipe-it#13794]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I use BitLocker.  When you set this up you get an automatically generated TXT file recovery key.  This is in UTF16LE encoding.
Snipe IT seems to not like this coding as it wont allow the file to be uploaded.  I need to change the encoding to UTF8.

### Reproduction steps

1. Create TXT file using UTF-16 LE encoding.
2. Upload this to a files section somewhere.
3.
...

### Expected behavior

To upload the file

### Snipe-IT Version

6.2.3

### Operating System

CentOs 7.9

### Web Server

Apache

### PHP Version

7.1

### Operating System

Windows 11

### Browser

edge

### Version

117

**Comment by snipe** [comment]
This seems like a pretty fringe case for what would result in a lot of workaround code. Is there anyone else running into this as well?

**Comment by jmarquez-ostcm** [comment]
Yes, but with offline domain join files created by Microsoft.  Tried adding a .gitattributes file to my repo to specify the file type extensions as CRLF or binary, but neither seem to have any effect.  

The problem I've seen is that the file uploaded to github and downloaded from github are slightly different.  Pre-upload the file contains about 5 less characters.  

Note: The text does include a =NUL at the end of it which seems to be confusing Github and appearing as a diamond with a ? mark in the middle of it.  When downloaded this introduces an extra line and
